src/components/ContactDetails/ContactDetailsAvatar.vue

```diff
@@ -133,6 +145,13 @@ export default {
 			}
 			return false
 		},
+		supportedSocial() {
```

Comment:
Please add a comment to describe what you're doing here, as this is a bit tricky :wink: